frontend/app/sysupport-admin/detail/controller.js

```diff
@@ -0,0 +1,118 @@
+import Controller from '@ember/controller'
+import { computed } from '@ember/object'
+import { inject as service } from '@ember/service'
+
+import { task } from 'ember-concurrency'
+import moment from 'moment'
+import { hash } from 'rsvp'
+import UIkit from 'UIkit'
+
+export default Controller.extend({
+  i18n: service(),
+  notify: service(),
+
+  duration: computed('validation.{hour,minute}', 'hour', 'minute', function() {
+    if (this.get('validation.hour') || this.get('validation.minute')) {
+      return moment.duration({
+        hours: this.get('validation.hour') && this.hour,
+        minutes: this.get('validation.minute') && this.minute
+      })
+    } else return null
+  }),
+
+  error: computed('validation.{hour,minute}', 'hour', 'minute', function() {
+    if (
+      (this.hour && !this.get('validation.hour')) ||
+      (this.minute && !this.get('validation.minute'))
+    ) {
+      return true
+    } else {
+      return false
+    }
+  }),
+
+  orders: computed('fetchModels.lastSuccessful.value', function() {
+    return (
+      this.get('fetchModels.lastSuccessful.value.orders') ||
+      this.get('model.orders')
+    )
+  }),
+
+  previewDuration: computed('duration', 'preview', function() {
+    return (
+      this.duration &&
+      this.preview &&
+      moment.duration(
+        this.get('project.purchasedTime') +
+          this.duration -
+          this.get('project.spentTime')
+      )
+    )
+  }),
+
+  project: computed('fetchModels.lastSuccessful.value', function() {
+    return (
+      this.get('fetchModels.lastSuccessful.value.project') ||
+      this.get('model.project')
+    )
+  }),
```

Comment:
Same here

Reply:
Why aren't you just using `fetchModels.lastSuccessful.value.project` in your template?